### ISSUE TITLE
Banking apps not working with this module installed 

### ISSUE THREAD

**Issue by ashoktvm** [symbuzzer/systemless-hosts-KernelSU-module#4]
As you already know the banking apps still not working with module installed. Hope you are still working on this

**Comment by symbuzzer** [comment]
> As you already know some banking apps still not working with module installed. Hope you are still working on this

At this stage, it does not seem possible to understand how these practitioners realize this. They're probably noticing that DNS requests are filtering through the hosts file, and that's how they detect it. But I'm not sure.

**Comment by hsantanna** [comment]
> 

From tests, the apps detect the module itself, not the hosts filtering.

I disable AdAway and reboot, but the apps still detect the module.

So I remove the module from KernelSU and reboot, and that way the apps won't detect anything wrong and starts to work.

**Comment by symbuzzer** [comment]
Can you uninstall this and try my Turkish Ad Hosts from my Github please?

**Comment by ashoktvm** [comment]
> Can you uninstall this and try my Turkish Ad Hosts from my Github please?

Adaway doesn't work with Turkish host installed

**Comment by symbuzzer** [comment]
No. Only check banking app please

**Comment by ashoktvm** [comment]
> No. Only check banking app please

It didnt created issue for Banking app. But the goal here is to work both adaway and Banking app right?

**Comment by symbuzzer** [comment]
> > No. Only check banking app please
> 
> It didnt created issue for Banking app. But the goal here is to work both adaway and Banking app right?
If you could confirm that the bank application works fine with TAH, will try to port its technic to this module. But I see that you find it more appropriate to question rather than help. 

I took note of your feedback. But at this stage, I can't help because there is no banking application that I can test for Turkey...

**Comment by ashoktvm** [comment]
> > > No. Only check banking app please
> > 
> > 
> > It didnt created issue for Banking app. But the goal here is to work both adaway and Banking app right?
> > If you could confirm that the bank application works fine with TAH, will try to port its technic to this module. But I see that you find it more appropriate to question rather than help.
> 
> I took note of your feedback. But at this stage, I can't help because there is no banking application that I can test for Turkey...

I was not questioning. But u were not clear to test it with a banking app or Adaway. Sorry if it felt like being rude

**Comment by symbuzzer** [comment]
@ashoktvm @hsantanna 
Can you try my other module, [adaway-kernelsu-helper](https://github.com/symbuzzer/adaway-kernelsu-helper)  and give feedback please?

One user, who I'm not sure what to do, reported that his device was bootlooped. However, I did not encounter such a problem in my own tests. If you do not know how to recover the device, do not try.

**Comment by ashoktvm** [comment]
> @ashoktvm @hsantanna Can you try my other module, [adaway-kernelsu-helper](https://github.com/symbuzzer/adaway-kernelsu-helper) and give feedback please?
> 
> One user, who I'm not sure what to do, reported that his device was bootlooped. However, I did not encounter such a problem in my own tests. If you do not know how to recover the device, do not try.

I was the one who tested it. Will try later. Cant afford to lose phone to bootloop now

**Comment by ashoktvm** [comment]
> @ashoktvm @hsantanna Can you try my other module, [adaway-kernelsu-helper](https://github.com/symbuzzer/adaway-kernelsu-helper) and give feedback please?
> 
> One user, who I'm not sure what to do, reported that his device was bootlooped. However, I did not encounter such a problem in my own tests. If you do not know how to recover the device, do not try.

Hi. 
I have tested the module. There was no reboot issue in my Redmi Note 11. But adaway was not able to install hosts file

**Comment by Lu5ck** [comment]
I second this, installing this expose rooting, making it detectable by banks / government apps etc.

**Comment by symbuzzer** [comment]
> I second this, installing this expose rooting, making it detectable by banks / government apps etc.

We already know this issue. But no solution for now. It is a cat-mouse game...

**Comment by Lu5ck** [comment]
> > I second this, installing this expose rooting, making it detectable by banks / government apps etc.
> 
> We already know this issue. But no solution for now. It is a cat-mouse game...

https://t.me/KernelSU_group/3249/44669

**Comment by Lu5ck** [comment]
> > I second this, installing this expose rooting, making it detectable by banks / government apps etc.
> 
> We already know this issue. But no solution for now. It is a cat-mouse game...

It seems like @HuskyDG already fixed this issue with
https://github.com/HuskyDG/ksu_unmount_injector

Just found out as I saw this posted in KSU telegram group

**Comment by adi90x** [comment]
Just try both adaway helper , and still detected by Microsoft Intune

**Comment by adi90x** [comment]
Hello, 

This module @gloeyisk/systemless-hosts is able to modify host without being detected by Intune app.
I guess detection is due to the overlay right ? 

Regards,

**Comment by symbuzzer** [comment]
> Hello,
> 
> This module @gloeyisk/systemless-hosts is able to modify host without being detected by Intune app. I guess detection is due to the overlay right ?
> 
> EDIT: Using @HuskyDG/ksu_unmount_injector adaway and Microsoft Intune works without any issue !
> 
> Regards,

Yes, even my [Turkish Ad Hosts] module does not cause any problems.

**Comment by symbuzzer** [comment]
Please read latest [README.md](https://github.com/symbuzzer/systemless-hosts-KernelSU-module#readme)
